File: beginners/vowel-counter/vowel-counter.py
def vowel_counter(string):
  vowel_count = 0
  vowels = 'aeiou'

  for char in string.lower():
    if vowels.find(char) != -1:
      vowel_count += 1
  
  return vowel_count

print(vowel_counter('anehizxcv'))

File: beginners/vowel-counter/vowel-counter.ts
function vowelCounter(word: string): number {
  let vowelCount = 0;
  const vowels = 'aeiou';

  word
    .toLowerCase()
    .split('')
    .forEach((char) => {
      if (vowels.indexOf(char) !== -1) {
        vowelCount++;
      }
    });

  return vowelCount;
}

console.log(vowelCounter('anehizxcv'));
